views/halloffame.html

```diff
@@ -69,6 +83,81 @@
 			</section >
 			<section class="hof-stats right-side">
 				<h2>My Stats</h2>
+				<div class="stats-helper">
+					<p><span class="regNum"></span> regular HOF conversations</p>
+					<p><span class="gpNum"></span> gold plated conversations</p>
+					<p><span class="gNum"></span> golden conversations</p>
+				</div>
+				<div id="chart"></div>
+			    <script>
+				    var $window = $(window);
+		
+			      	(function(d3) {
+			        'use strict';
+
+				        var dataset = [
+				          { label: 'gold', count: goldNum }, 
+				          { label: 'goldPlated', count: goldPlatedNum },
+				          { label: 'regHof', count: regHofNum }
+				        ];
+
+				        var width = 142;
+				        var height = 142;
+				        var donutWidth = 20;
+
+				        // reize svg for responsive screens;
+				        function checkWidth() {
+					        var windowsize = $window.width();
+					        if (windowsize > 820 && windowsize < 1190) {
+					            width = 90;
+					            height = 90;
+					            donutWidth = 12;
+					            $('#chart').css({
+					            	marginLeft: '1.5rem'
+					            })
+					        } else {
+					        	width = 145	;
+				        		height = 145;
+				        		donutWidth = 20;
+				        		$('#chart').css({
+					            	marginLeft: '-1rem'
+					            })
+					        }
+					    }
+
+					    checkWidth();
+					    $(window).resize(checkWidth);
```

Comment:
On the line before (128), the checkWidth() function is called to set the appropriate width/height/etc variables before the circles are created.  This line (129) seems to tell the browser to run the checkWidth() function every time the browser window is resized.  However, the checkWidth function doesn't seem to re-draw the arcs after the new dimensions are set.  In effect, the arc size doesn't seem to change at all when I resize the window, even though it seems like that is the function's purpose.

If you'd like to have them resize themselves when the browser is resized, you would need to make a function that clears and re-draws the arcs, and place this function inside checkWidth().

However, on my screen at least, the difference in size isn't substantial.  I think it would look fine with either of the dimensions.  If you don't want 2 different sizes, you could simply remove this line 129.  It's up to you!